packages/core/src/controllers/ConnectionController.ts

```diff
@@ -247,5 +252,48 @@ export const ConnectionController = {
     } catch (error) {
       throw new Error('Failed to disconnect')
     }
+  },
+
+  /**
+   * @experimental - This is an experimental feature and may be subject to change.
+   * Initializes SIWX if available.
+   * This is not yet considering One Click Auth.
+   */
+  async initializeSWIXIfAvailable() {
+    const swix = OptionsController.state.siwx
+    if (!swix) {
+      return
+    }
+
+    if (OptionsController.state.isSiweEnabled) {
```

Comment:
We are setting the `isSiweEnabled` value in `appkit/client` while we initializing the controllers. Can we double sure that this is not called before it?

Reply:
This only going to be called during AppKit connection, so after the AppKit is initialized.